app/kubernetes/views/applications/create/createApplication.html

```diff
@@ -226,7 +227,7 @@
                 <div ng-if="ctrl.formValues.ResourcePool">
                   <!-- #region STACK -->
                   <kube-stack-name
-                    ng-if="ctrl.state.appType !== ctrl.KubernetesDeploymentTypes.APPLICATION_FORM"
+                    ng-if="!ctrl.deploymentOptions.hideStacksFunctionality"
```

Comment:
Is there meant to be a text-tip in here like in EE?

Reply:
Updated